core/src/main/java/jenkins/model/DownloadSettings.java

```diff
@@ -134,18 +109,4 @@ public DailyCheck() {
         }
 
     }
-
-    @Extension public static final class Warning extends AdministrativeMonitor {
```

Comment:
Forgot to delete `core/src/main/resources/jenkins/model/DownloadSettings/Warning/`.